examples/storm-starter/pom.xml

```diff
@@ -96,6 +96,12 @@
       <groupId>com.google.guava</groupId>
       <artifactId>guava</artifactId>
     </dependency>
+    <dependency>
+      <groupId>org.apache.storm</groupId>
+      <artifactId>storm-kafka</artifactId>
```

Comment:
should move ``` <module>examples/storm-starter</module>``` in [pom.xml](https://github.com/apache/storm/blob/master/pom.xml#L164) after ```<module>external/flux</module>```